build-plugin/src/main/kotlin/SpotlessExtension.kt

```diff
@@ -40,7 +40,11 @@ fun SpotlessExtension.configureMarkdownCheck(
     targets: List<String>,
 ) {
     format("markdown") {
-        prettier()
+        prettier("3.3.3").config(
```

Comment:
Is there any evidence that pinning the version to 3.3.3 fixes an issue? If so, we should add a comment explaining why this particular version was picked. If not, I think it's best to avoid explicitly specifying a version.

Reply:
Why should we avoid specifying a version?

Reply:
Because then we'll have to manage updates ourselves. In practice this probably means we'll keep using this version until spotless drops support for it and we're forced to deal with a potentially massive upgrade. In the meantime we might miss out on improvements we'd otherwise get for free.

Reply:
For the sake of people reading this code/config in the future, please add a comment explaining why a version was specified (instead of using the Spotless default).

Reply:
I believe the debate over pinning the Prettier version is unnecessary for this context. The feature itself is simple and does not warrant the complexity of auto-updates, which are challenging to implement and maintain. Since formatting tools like Prettier rarely cause critical issues in minor updates, it's more pragmatic to prioritize resolving the task rather than over-optimizing version management. Let’s focus on merging the fix and address potential tooling updates when they become relevant. Thoughts?

Reply:
SGTM. If you can add a short comment explaining why we're pinning a specific version I believe we can get this merged quickly.